src/components/productoverview/ProductList.vue

```diff
@@ -104,6 +122,14 @@ export default {
     changeSort(sort) {
       this.sort = sort;
     },
+
+    hasManyProducts() {
+      return this.products?.results.length >= this.limit / 2;
+    },
```

Comment:
Is this a method or a computed property? (I'm aware that it's partly my fault it is here :P)

Reply:
haha noway! of course it's a computed property, thanks for noticing